modules/quickstart-sap/pages/configure.adoc

```diff
@@ -2,7 +2,7 @@
 = Configure Clients
 
 {productname} uses formulas with forms to configure your SAP clients.
-There are two formulas that you need to use: ``habootstrap`` to configure the HANA database, and ``FIXME`` to configure the clients into a cluster.
+There are two formulas that you need to use: ``Hana`` to configure the HANA database, and ``Cluster`` to configure the clients into a cluster.
 
 The formulas are provided by packages that you can download from your package manager.
```

Comment:
Drive-by remark: I think you can download the packages _with_ the package manager (I always think about it as a software tool) or _from_ an installation source (repo, etc.). Maybe, I'm blinded by my German language background, though.

Reply:
No, I think that's pretty reasonable, actually. I'll update.